libs/lawnchair/lawnchairHtml5FileSystem.js

```diff
@@ -1,25 +1,23 @@
 Lawnchair.adapter('html5-filesystem', (function(global){
-
-  var FileError = global.FileError;
-
+  
   var fail = function( e ) {
     var msg;
     var show = true;
-    switch (e.code) {
-      case FileError.QUOTA_EXCEEDED_ERR:
+    switch (e.name) {
+      case "QuotaExceededError":
         msg = 'QUOTA_EXCEEDED_ERR';
         break;
-      case FileError.NOT_FOUND_ERR:
+      case "NotFoundError":
         msg = 'NOT_FOUND_ERR';
         show = false;
         break;
-      case FileError.SECURITY_ERR:
+      case "SecurityError":
         msg = 'SECURITY_ERR';
         break;
-      case FileError.INVALID_MODIFICATION_ERR:
+      case "InvalidModificationError":
         msg = 'INVALID_MODIFICATION_ERR';
         break;
-      case FileError.INVALID_STATE_ERR:
+      case "InvalidStateError":
```

Comment:
I spent some time to investigate how to do it to cover most of the browsers. FileError is really old - it was depreciated couple years ago.  This code would not break anything as we just use it to log message. In previous version of the code it was throwing error and stoping entire javascript thread from processing. 
We using this only to log actual message which is carrying the same information as actual error.
I think we can even simplify this (going to add another commit)